### PR TITLE
Fixed framerate in the player defaults to the framerate in the series

### PR DESCRIPTION
**Pull request by horner**
Screen Shot 2022-08-21 at 11 03 28 AM

### CODE REVIEW

**Comment by horner** [comment]
Watch https://www.youtube.com/watch?v=QyyXQaXOWr0

image

File: platform/viewer/src/connectedComponents/ConnectedStudyBrowser.js [hunk 1]
```diff
@@ -90,8 +91,18 @@ const mapDispatchToProps = (dispatch, ownProps) => {
           type: 'error',
         });
       }
-
       dispatch(setActiveViewportSpecificData(displaySet));
+      //console.log('wwwwwwwwwwwwwwwwwwwwwwwwwww ', displaySet);
```
Comment (igoroctaviano) [inline platform/viewer/src/connectedComponents/ConnectedStudyBrowser.js:95]: Can you please remove this comment?

File: platform/core/src/classes/StudyPrefetcher.js [hunk 2]
```diff
@@ -204,6 +204,7 @@ export class StudyPrefetcher {
       'SeriesInstanceUID',
       image.imageId
     );
+    if (!study) return null;
```
Comment (igoroctaviano) [inline platform/core/src/classes/StudyPrefetcher.js:207]: I'm wondering what getSeriesByUID and getInstanceByUID return if not found? maybe it already returns null?

**Review by igoroctaviano — APPROVED** [review]
LGTM 🎖️ 
*(left two optional comments)

**Comment by sedghi** [comment]
base has changed, read more here https://github.com/OHIF/Viewers/issues/3477

**Comment by sedghi** [comment]
we should investigate this in v3

**Comment by jbocce** [comment]
This PR is being closed in favour of this new PR https://github.com/OHIF/Viewers/pull/3735 which targets the most current master branch. Thanks @horner  and @rareramos for the inspiration and ideas.

**Pull request closed without merging**